src/tasks/helpers/getContainer.js

```diff
@@ -96,23 +98,31 @@ var augmentModule = function(modulesOutput, extensionName, extensionPath, module
   });
 };
 
-var augmentModules = function(extensionOutput, extensionDescriptor, extensionPath) {
+const augmentModules = function(extensionOutput, extensionDescriptor, extensionPath) {
   extensionOutput.modules = extensionOutput.modules || {};
 
   FEATURE_TYPES.forEach(function(featureType) {
     if (extensionDescriptor.hasOwnProperty(featureType)) {
-      var featureDescriptors = extensionDescriptor[featureType];
+      let featureDescriptors = flatten([extensionDescriptor[featureType]]);
+
+      if (typeof featureDescriptors === 'string') {
+        featureDescriptors = [{
+          libPath: featureDescriptors
+        }];
+      }
 
       if (featureDescriptors) {
         featureDescriptors.forEach(function(featureDescriptor) {
-          var modulePath = path.join(
+          const modulePath = path.join(
             extensionPath,
             extensionDescriptor.libBasePath || '',
-            featureDescriptor.libPath);
+            featureDescriptor.libPath || featureDescriptor);
```

Comment:
Now `featureDescriptor.libPath || featureDescriptor);` should change to `featureDescriptor.libPath);` I think.